src/Forms/GridField/GridFieldDetailForm_ItemRequest.php

```diff
@@ -800,6 +816,7 @@ public function doDelete($data, $form)
         //when an item is deleted, redirect to the parent controller
         $controller = $this->getToplevelController();
         $controller->getRequest()->addHeader('X-Pjax', 'Content'); // Force a content refresh
+        $controller->getRequest()->addHeader('X-Status', $message);
```

Comment:
See if we still need this after the change I've requested. Maybe we can remove this.

Reply:
Updated

Reply:
Updated